PROJECT

```diff
@@ -6,4 +6,13 @@ plugins:
   scorecard.sdk.operatorframework.io/v2: {}
 projectName: osbuild-operator
 repo: github.com/project-flotta/osbuild-operator
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
```

Comment:
Do we want to allow many (different namespaces) `OSBuildConfig` in the cluster?

Reply:
Yes, same as for BuildConfig. This will allow access control over the CRs